lib/n_plus_one_control/rspec/matchers/perform_linear_number_of_queries.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# rubocop:disable  Metrics/BlockLength
+::RSpec::Matchers.define :perform_linear_number_of_queries do |slope: 1|
```

Comment:
I'm not sure if we need to provide a default value on `slope`. What do you think?

Reply:
I think, the default of one makes sense since it's literally "N+1" queries :)